src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -974,7 +974,7 @@ define([
          */
         toggleDescriptions: function() {
             var checked = this.sandbox.dom.is(
-                    this.sandbox.dom.find(constants.descriptionCheckboxSelector, this.$multiple), ':checked'
```

Comment:
i think this indentation is correct. could you please reset this change

Reply:
Done